cns/NetworkContainerContract.go

```diff
@@ -46,6 +51,12 @@ type CreateNetworkContainerRequest struct {
 	Routes                     []Route
 }
 
+// ConfigureNetworkContainerNetworkingRequest - specifies request to attach/detach network container to network.
+type ConfigureNetworkContainerNetworkingRequest struct {
+	DockerContainerid string
```

Comment:
I caps

Reply:
Following the style from CreateNetworkContainerRequest struct